neurom/analysis/morphtree.py

```diff
@@ -222,8 +222,8 @@ def partition(tree):
     '''
     def partition_at_point(bif_point):
         '''Partition at each bif point.'''
-        n = n_sections(bif_point.children[0])
-        m = n_sections(bif_point.children[1])
+        n = float(n_sections(bif_point.children[0]))
```

Comment:
The test was not failing because the division was with 1, giving the correct result. I added the partition in the bifurcations file, and added a general test that does fail in the case of integer division.